### ISSUE TITLE
Process fault/crash and refused to resume

### ISSUE THREAD

**Issue by effofxprime** [aura-nw/aura#23]
I woke up this morning with my validator being down.  After I inspected, only peer gossip was being shown in the log.  I restarted the service and got this:

```
Aug 24 03:52:26 horace systemd[1]: Starting Aura Validator...
Aug 24 03:52:26 horace systemd[1]: Started Aura Validator.
Aug 24 03:52:26 horace aurad[26177]: 3:52AM INF starting node with ABCI Tendermint in-process
Aug 24 03:52:26 horace aurad[26177]: 3:52AM INF Starting multiAppConn service impl=multiAppConn module=proxy
Aug 24 03:52:26 horace aurad[26177]: 3:52AM INF Starting localClient service connection=query impl=localClient module=abci-client
Aug 24 03:52:26 horace aurad[26177]: 3:52AM INF Starting localClient service connection=snapshot impl=localClient module=abci-client
Aug 24 03:52:26 horace aurad[26177]: 3:52AM INF Starting localClient service connection=mempool impl=localClient module=abci-client
Aug 24 03:52:26 horace aurad[26177]: 3:52AM INF Starting localClient service connection=consensus impl=localClient module=abci-client
Aug 24 03:52:26 horace aurad[26177]: 3:52AM INF Starting EventBus service impl=EventBus module=events
Aug 24 03:52:26 horace aurad[26177]: 3:52AM INF Starting PubSub service impl=PubSub module=pubsub
Aug 24 03:52:26 horace aurad[26177]: 3:52AM INF Starting IndexerService service impl=IndexerService module=txindex
Aug 24 03:52:26 horace aurad[26177]: 3:52AM INF Found local state with non-zero height, skipping state sync
Aug 24 03:52:26 horace aurad[26177]: 3:52AM INF ABCI Handshake App Info hash="-/E�ج�9.��\x14�}ޜo \x01Z\x17|���\v'I\x0e8�\x0f" height=629134 module=consensus protocol-version=0 software-version=euphoria_4027003
Aug 24 03:52:26 horace aurad[26177]: 3:52AM INF ABCI Replay Blocks appHeight=629134 module=consensus stateHeight=629134 storeHeight=629135
Aug 24 03:52:26 horace aurad[26177]: 3:52AM INF Replay last block using real app module=consensus
Aug 24 03:52:26 horace aurad[26177]: panic: runtime error: invalid memory address or nil pointer dereference
Aug 24 03:52:26 horace aurad[26177]: [signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x17aedb3]
Aug 24 03:52:26 horace aurad[26177]: goroutine 1 [running]:
Aug 24 03:52:26 horace aurad[26177]: github.com/cosmos/cosmos-sdk/server.startInProcess(_, {{0x0, 0x0, 0x0}, {0x25da6d8, 0xc0004d0a80}, {0xc000365630, 0xa}, {0x25c4be0, 0xc0001a0510}, ...}, ...)
Aug 24 03:52:26 horace aurad[26177]:         /validator-home-1/aura/go/pkg/mod/github.com/cosmos/cosmos-sdk@v0.45.2/server/start.go:283 +0x813
Aug 24 03:52:26 horace aurad[26177]: github.com/cosmos/cosmos-sdk/server.StartCmd.func2(0xc000ea4780?, {0x33bf668?, 0x0?, 0x0?})
Aug 24 03:52:26 horace aurad[26177]:         /validator-home-1/aura/go/pkg/mod/github.com/cosmos/cosmos-sdk@v0.45.2/server/start.go:126 +0x169
Aug 24 03:52:26 horace aurad[26177]: github.com/spf13/cobra.(*Command).execute(0xc000ea4780, {0x33bf668, 0x0, 0x0})
Aug 24 03:52:26 horace aurad[26177]:         /validator-home-1/aura/go/pkg/mod/github.com/spf13/cobra@v1.3.0/command.go:856 +0x67c
Aug 24 03:52:26 horace aurad[26177]: github.com/spf13/cobra.(*Command).ExecuteC(0xc000db2280)
Aug 24 03:52:26 horace aurad[26177]:         /validator-home-1/aura/go/pkg/mod/github.com/spf13/cobra@v1.3.0/command.go:974 +0x3bd
Aug 24 03:52:26 horace aurad[26177]: github.com/spf13/cobra.(*Command).Execute(...)
Aug 24 03:52:26 horace aurad[26177]:         /validator-home-1/aura/go/pkg/mod/github.com/spf13/cobra@v1.3.0/command.go:902
Aug 24 03:52:26 horace aurad[26177]: github.com/spf13/cobra.(*Command).ExecuteContext(...)
Aug 24 03:52:26 horace aurad[26177]:         /validator-home-1/aura/go/pkg/mod/github.com/spf13/cobra@v1.3.0/command.go:895
Aug 24 03:52:26 horace aurad[26177]: github.com/cosmos/cosmos-sdk/server/cmd.Execute(0xc000db2280?, {0xc00005a880, 0x1c})
Aug 24 03:52:26 horace aurad[26177]:         /validator-home-1/aura/go/pkg/mod/github.com/cosmos/cosmos-sdk@v0.45.2/server/cmd/execute.go:36 +0x1eb
Aug 24 03:52:26 horace aurad[26177]: main.main()
Aug 24 03:52:26 horace aurad[26177]:         /validator-home-1/aura/aura/cmd/aurad/main.go:44 +0x126
Aug 24 03:52:26 horace systemd[1]: aura.service: Main process exited, code=exited, status=2/INVALIDARGUMENT
Aug 24 03:52:26 horace systemd[1]: aura.service: Failed with result 'exit-code'.
```

To resolve the issue I performed an `unsafe-reset-all` and did a fresh state sync.
Any idea on what may have initially caused it?   I run several nodes on one server.  This was the only one to have an issue.  I would expect to see more of the nodes down if it was a drive or some other type of possible hardware issue.

**Comment by effofxprime** [comment]
I should have also posted this earlier, but it was early in the AM for me when I discovered this and forgot to check the log at the time of incident.

```
 Aug 24 01:57:29 horace aurad[1375]: 1:57AM INF indexed block height=629133 module=txindex
21310 Aug 24 01:57:34 horace aurad[1375]: 1:57AM INF Timed out dur=4956.079793 height=629134 module=consensus round=0 step=1
21311 Aug 24 01:57:34 horace aurad[1375]: 1:57AM INF received proposal module=consensus proposal={"Type":32,"block_id":{"hash":"0AA
      42E2CE353BEEEC227C7C75AADDB5113697D71496EEA0B23B82C648A4079A1","parts":{"hash":"135B5DAF59FB57827886521FF4E49ABF9BC3BAF7BE3C6
      5075A9727B6B001E1DD","total":1}},"height":629134,"pol_round":-1,"round":0,"signature":"Q317GMcszsTI3P+3gBZWbG4ub8AWM5kA1+hugj
      87hdNXjwA/dNtDToW4FKAbWxMfP8DQlMDPn1V2DnrlLBPbAw==","timestamp":"2022-08-24T07:57:34.574386783Z"}
21312 Aug 24 01:57:34 horace aurad[1375]: 1:57AM INF received complete proposal block hash=0AA42E2CE353BEEEC227C7C75AADDB5113697D71
      496EEA0B23B82C648A4079A1 height=629134 module=consensus
21313 Aug 24 01:57:35 horace aurad[1375]: 1:57AM INF finalizing commit of block hash=0AA42E2CE353BEEEC227C7C75AADDB5113697D71496EEA
      0B23B82C648A4079A1 height=629134 module=consensus num_txs=1 root=249B978B8D250125CED9B29819D7C5CA67E376D2EA1772CAAACF5FA963B8
      5B6B
21314 Aug 24 01:57:35 horace aurad[1375]: 1:57AM INF Get max supply from aura maxSupply=1000000000000000 module=x/mint
21315 Aug 24 01:57:35 horace aurad[1375]: 1:57AM INF Get current supply from network currentSupply=754829147592497 module=x/mint
21316 Aug 24 01:57:35 horace aurad[1375]: 1:57AM INF Exclude Addr exclude_addr=299791099550000ueaura module=x/mint
21317 Aug 24 01:57:35 horace aurad[1375]: 1:57AM INF Value BondedRatio:  bondedRatio=0.010400336409532542 module=x/mint
21318 Aug 24 01:57:35 horace aurad[1375]: 1:57AM INF minted coins from module account amount=7485620ueaura from=mint module=x/bank
21319 Aug 24 01:57:35 horace aurad[1375]: 1:57AM INF executed block height=629134 module=state num_invalid_txs=1 num_valid_txs=0
21320 Aug 24 01:57:35 horace aurad[1375]: 1:57AM INF commit synced commit=436F6D6D697449447B5B3435203437203639203138342032313620313
      73220313530203537203436203233312032343920323020313937203132352032323220313536203131312033322031203930203233203132342031373020
      31353020313334203131203339203733203134203536203136332031355D3A39393938457D
21321 Aug 24 01:57:35 horace aurad[1375]: 1:57AM INF committed state app_hash=2D2F45B8D8AC96392EE7F914C57DDE9C6F20015A177CAA96860B2
      7490E38A30F height=629134 module=state num_txs=1
21322 Aug 24 01:57:35 horace aurad[1375]: 1:57AM INF indexed block height=629134 module=txindex
21323 Aug 24 01:57:40 horace aurad[1375]: 1:57AM INF Timed out dur=4973.65598 height=629135 module=consensus round=0 step=1
21324 Aug 24 01:57:40 horace aurad[1375]: 1:57AM INF received proposal module=consensus proposal={"Type":32,"block_id":{"hash":"89D
      EC836843A34F82BBEC0F7BF44AB8B3E5A3401C034C919E6557CE538F976A9","parts":{"hash":"74C6E3686B6D4C1EA1EB9A8846AE63D0AE355733CBDBB
      C1101B0229AA79473CF","total":1}},"height":629135,"pol_round":-1,"round":0,"signature":"MADwkaBenCnjwIHD6WZJl44t2s+gaXI0N7Fm3c
      ppDOCVBG8VvwHSLvaai0PsojwkPTmyPWIeudls/61voiMsBA==","timestamp":"2022-08-24T07:57:40.237750332Z"}
21325 Aug 24 01:57:40 horace aurad[1375]: 1:57AM INF received complete proposal block hash=89DEC836843A34F82BBEC0F7BF44AB8B3E5A3401
      C034C919E6557CE538F976A9 height=629135 module=consensus
21326 Aug 24 01:57:40 horace aurad[1375]: 1:57AM ERR prevote step: ProposalBlock is invalid err="wrong Block.Header.AppHash.  Expec
      ted 2D2F45B8D8AC96392EE7F914C57DDE9C6F20015A177CAA96860B27490E38A30F, got CA138707FAA8D357C7C967FDC82A40A0D4398B990933EBED524
      C24CDB5EFCF75" height=629135 module=consensus round=0
21327 Aug 24 01:57:40 horace aurad[1375]: 1:57AM ERR CONSENSUS FAILURE!!! err="precommit step; +2/3 prevoted for an invalid block:
      wrong Block.Header.AppHash.  Expected 2D2F45B8D8AC96392EE7F914C57DDE9C6F20015A177CAA96860B27490E38A30F, got CA138707FAA8D357C
      7C967FDC82A40A0D4398B990933EBED524C24CDB5EFCF75" module=consensus stack="goroutine 67487 [running]:\nruntime/debug.Stack()\n\
      t/usr/local/go/src/runtime/debug/stack.go:24 +0x65\ngithub.com/tendermint/tendermint/consensus.(*State).receiveRoutine.func2(
      )\n\t/validator-home-1/aura/go/pkg/mod/github.com/tendermint/tendermint@v0.34.16/consensus/state.go:727 +0x4c\npanic({0x18fdf
      e0, 0xc01c2bc5c0})\n\t/usr/local/go/src/runtime/panic.go:884 +0x212\ngithub.com/tendermint/tendermint/consensus.(*State).ente
      rPrecommit(0xc0000a6000, 0x9998f, 0x0)\n\t/validator-home-1/aura/go/pkg/mod/github.com/tendermint/tendermint@v0.34.16/consens
      us/state.go:1391 +0x1679\ngithub.com/tendermint/tendermint/consensus.(*State).addVote(0xc0000a6000, 0xc0202a79a0, {0xc000da1e
      90, 0x28})\n\t/validator-home-1/aura/go/pkg/mod/github.com/tendermint/tendermint@v0.34.16/consensus/state.go:2107 +0x13ef\ngi
      thub.com/tendermint/tendermint/consensus.(*State).tryAddVote(0xc0000a6000, 0xc0202a79a0, {0xc000da1e90?, 0xc012fdee00?})\n\t/
      validator-home-1/aura/go/pkg/mod/github.com/tendermint/tendermint@v0.34.16/consensus/state.go:1933 +0x2c\ngithub.com/tendermi
      nt/tendermint/consensus.(*State).handleMsg(0xc0000a6000, {{0x25a7580?, 0xc00ce876b8?}, {0xc000da1e90?, 0x0?}})\n\t/validator-
      home-1/aura/go/pkg/mod/github.com/tendermint/tendermint@v0.34.16/consensus/state.go:839 +0x170\ngithub.com/tendermint/tenderm
      int/consensus.(*State).receiveRoutine(0xc0000a6000, 0x0)\n\t/validator-home-1/aura/go/pkg/mod/github.com/tendermint/tendermin
      t@v0.34.16/consensus/state.go:763 +0x3f9\ncreated by github.com/tendermint/tendermint/consensus.(*State).OnStart\n\t/validato
      r-home-1/aura/go/pkg/mod/github.com/tendermint/tendermint@v0.34.16/consensus/state.go:379 +0x11c\n"
21328 Aug 24 01:57:40 horace aurad[1375]: 1:57AM INF Stopping baseWAL service impl={"Logger":{}} module=consensus wal=/validator-ho
      me-1/aura/.aura/data/cs.wal/wal
21329 Aug 24 01:57:40 horace aurad[1375]: 1:57AM INF Stopping Group service impl={"Dir":"/validator-home-1/aura/.aura/data/cs.wal",
      "Head":{"ID":"yNMZayuCuFxw:/validator-home-1/aura/.aura/data/cs.wal/wal","Path":"/validator-home-1/aura/.aura/data/cs.wal/wal
      "},"ID":"group:yNMZayuCuFxw:/validator-home-1/aura/.aura/data/cs.wal/wal","Logger":{}} module=consensus wal=/validator-home-1
      /aura/.aura/data/cs.wal/wal
21330 Aug 24 01:57:41 horace aurad[1375]: 1:57AM INF Timed out dur=1000 height=629135 module=consensus round=0 step=5
```

**Comment by effofxprime** [comment]
Im not sure if it's the contract causing the issue.  The contract is on 2 blocks before.  For this current log, the block I failed to sign and come to consensus to was 640815.  The instantiate contract was block 640813.

```
 Aug 24 20:38:15 horace aurad[26995]: 8:38PM ERR prevote step: ProposalBlock is invalid err="wrong Block.Header.AppHash.  Exp
       ected 3386B2D8D2EC7B3A317B939131A08261A5954B799272EE1A4D88197F38D74F88, got 5319D082BDC65323062528FEBA5E55997B4D1E5248A73573
       02C63E410EA63C27" height=640814 module=consensus round=0
218442 Aug 24 20:38:15 horace aurad[26995]: 8:38PM INF Connection is closed @ recvRoutine (likely by the other side) conn={"Logger"
       :{}} module=p2p peer={"id":"705e3c2b2b554586976ed88bb27f68e4c4176a33","ip":"13.250.223.114","port":26656}
218443 Aug 24 20:38:15 horace aurad[26995]: 8:38PM INF Stopping MConnection service impl={"Logger":{}} module=p2p peer={"id":"705e3
       c2b2b554586976ed88bb27f68e4c4176a33","ip":"13.250.223.114","port":26656}
218444 Aug 24 20:38:15 horace aurad[26995]: 8:38PM ERR Stopping peer for error err=EOF module=p2p peer={"Data":{},"Logger":{}}
218445 Aug 24 20:38:15 horace aurad[26995]: 8:38PM INF Stopping Peer service impl={"Data":{},"Logger":{}} module=p2p peer={"id":"70
       5e3c2b2b554586976ed88bb27f68e4c4176a33","ip":"13.250.223.114","port":26656}
218446 Aug 24 20:38:16 horace aurad[26995]: 8:38PM INF Connection is closed @ recvRoutine (likely by the other side) conn={"Logger"
       :{}} module=p2p peer={"id":"b9243524f659f2ff56691a4b2919c3060b2bb824","ip":"13.214.5.1","port":26656}
218447 Aug 24 20:38:16 horace aurad[26995]: 8:38PM INF Stopping MConnection service impl={"Logger":{}} module=p2p peer={"id":"b9243
       524f659f2ff56691a4b2919c3060b2bb824","ip":"13.214.5.1","port":26656}
218448 Aug 24 20:38:16 horace aurad[26995]: 8:38PM ERR Stopping peer for error err=EOF module=p2p peer={"Data":{},"Logger":{}}
218449 Aug 24 20:38:16 horace aurad[26995]: 8:38PM INF Stopping Peer service impl={"Data":{},"Logger":{}} module=p2p peer={"id":"b9
       243524f659f2ff56691a4b2919c3060b2bb824","ip":"13.214.5.1","port":26656}
218450 Aug 24 20:38:16 horace aurad[26995]: 8:38PM ERR CONSENSUS FAILURE!!! err="precommit step; +2/3 prevoted for an invalid block
       : wrong Block.Header.AppHash.  Expected 3386B2D8D2EC7B3A317B939131A08261A5954B799272EE1A4D88197F38D74F88, got 5319D082BDC653
       23062528FEBA5E55997B4D1E5248A7357302C63E410EA63C27" module=consensus stack="goroutine 9190 [running]:\nruntime/debug.Stack()
       \n\t/usr/local/go/src/runtime/debug/stack.go:24 +0x65\ngithub.com/tendermint/tendermint/consensus.(*State).receiveRoutine.fu
       nc2()\n\t/validator-home-1/aura/go/pkg/mod/github.com/tendermint/tendermint@v0.34.16/consensus/state.go:727 +0x4c\npanic({0x
       18fdfe0, 0xc00c57a070})\n\t/usr/local/go/src/runtime/panic.go:884 +0x212\ngithub.com/tendermint/tendermint/consensus.(*State
       ).enterPrecommit(0xc000e55880, 0x9c72e, 0x0)\n\t/validator-home-1/aura/go/pkg/mod/github.com/tendermint/tendermint@v0.34.16/
       consensus/state.go:1391 +0x1679\ngithub.com/tendermint/tendermint/consensus.(*State).addVote(0xc000e55880, 0xc01140f220, {0x
       c00305e390, 0x28})\n\t/validator-home-1/aura/go/pkg/mod/github.com/tendermint/tendermint@v0.34.16/consensus/state.go:2107 +0
       x13ef\ngithub.com/tendermint/tendermint/consensus.(*State).tryAddVote(0xc000e55880, 0xc01140f220, {0xc00305e390?, 0xc0091a00
       00?})\n\t/validator-home-1/aura/go/pkg/mod/github.com/tendermint/tendermint@v0.34.16/consensus/state.go:1933 +0x2c\ngithub.c
       om/tendermint/tendermint/consensus.(*State).handleMsg(0xc000e55880, {{0x25a7580?, 0xc00dceca48?}, {0xc00305e390?, 0x0?}})\n\
       t/validator-home-1/aura/go/pkg/mod/github.com/tendermint/tendermint@v0.34.16/consensus/state.go:839 +0x170\ngithub.com/tende
       rmint/tendermint/consensus.(*State).receiveRoutine(0xc000e55880, 0x0)\n\t/validator-home-1/aura/go/pkg/mod/github.com/tender
       mint/tendermint@v0.34.16/consensus/state.go:763 +0x3f9\ncreated by github.com/tendermint/tendermint/consensus.(*State).OnSta
       rt\n\t/validator-home-1/aura/go/pkg/mod/github.com/tendermint/tendermint@v0.34.16/consensus/state.go:379 +0x11c\n"
218451 Aug 24 20:38:16 horace aurad[26995]: 8:38PM INF Stopping baseWAL service impl={"Logger":{}} module=consensus wal=/validator-
       home-1/aura/.aura/data/cs.wal/wal
218452 Aug 24 20:38:16 horace aurad[26995]: 8:38PM INF Stopping Group service impl={"Dir":"/validator-home-1/aura/.aura/data/cs.wal
       ","Head":{"ID":"LGhGfKrhxY5w:/validator-home-1/aura/.aura/data/cs.wal/wal","Path":"/validator-home-1/aura/.aura/data/cs.wal/
       wal"},"ID":"group:LGhGfKrhxY5w:/validator-home-1/aura/.aura/data/cs.wal/wal","Logger":{}} module=consensus wal=/validator-ho
       me-1/aura/.aura/data/cs.wal/wal
```

**Comment by effofxprime** [comment]
Server/Go version
```
aura@horace:~/.aura$ go version
go version go1.19 linux/amd64
aura@horace:~/.aura$ uname -r
5.15.0-46-generic
```

Node version
```
aura@horace:~/.aura$ aurad version --long
name: aura
server_name: <appd>
version: euphoria_4027003
commit: 402700319364b72477a182508f4ff12c3f8dbc3d
build_tags: ""
go: go version go1.19 linux/amd64
build_deps:
- filippo.io/edwards25519@v1.0.0-beta.2
- github.com/99designs/keyring@v1.1.6 => github.com/cosmos/keyring@v1.1.7-0.20210622111912-ef00f8ac3d76
- github.com/ChainSafe/go-schnorrkel@v0.0.0-20200405005733-88cbf1b4c40d
- github.com/CosmWasm/wasmd@v0.24.0
- github.com/CosmWasm/wasmvm@v1.0.0-beta7
- github.com/Workiva/go-datastructures@v1.0.53
- github.com/armon/go-metrics@v0.3.10
- github.com/beorn7/perks@v1.0.1
- github.com/bgentry/speakeasy@v0.1.0
- github.com/btcsuite/btcd@v0.22.0-beta
- github.com/cespare/xxhash/v2@v2.1.2
- github.com/coinbase/rosetta-sdk-go@v0.7.0
- github.com/confio/ics23/go@v0.7.0
- github.com/cosmos/btcutil@v1.0.4
- github.com/cosmos/cosmos-sdk@v0.45.2
- github.com/cosmos/go-bip39@v1.0.0
- github.com/cosmos/iavl@v0.17.3
- github.com/cosmos/ibc-go/v2@v2.2.0
- github.com/davecgh/go-spew@v1.1.1
- github.com/desertbit/timer@v0.0.0-20180107155436-c41aec40b27f
- github.com/dvsekhvalnov/jose2go@v0.0.0-20200901110807-248326c1351b
- github.com/felixge/httpsnoop@v1.0.1
- github.com/fsnotify/fsnotify@v1.5.1
- github.com/go-kit/kit@v0.12.0
- github.com/go-kit/log@v0.2.0
- github.com/go-logfmt/logfmt@v0.5.1
- github.com/godbus/dbus@v0.0.0-20190726142602-4481cbc300e2
- github.com/gogo/gateway@v1.1.0
- github.com/gogo/protobuf@v1.3.3 => github.com/regen-network/protobuf@v1.3.3-alpha.regen.1
- github.com/golang/protobuf@v1.5.2
- github.com/golang/snappy@v0.0.3
- github.com/google/btree@v1.0.0
- github.com/google/gofuzz@v1.2.0
- github.com/google/orderedcode@v0.0.1
- github.com/gorilla/handlers@v1.5.1
- github.com/gorilla/mux@v1.8.0
- github.com/gorilla/websocket@v1.5.0
- github.com/grpc-ecosystem/go-grpc-middleware@v1.3.0
- github.com/grpc-ecosystem/grpc-gateway@v1.16.0
- github.com/gsterjov/go-libsecret@v0.0.0-20161001094733-a6f4afe4910c
- github.com/gtank/merlin@v0.1.1
- github.com/gtank/ristretto255@v0.1.2
- github.com/hashicorp/go-immutable-radix@v1.3.1
- github.com/hashicorp/golang-lru@v0.5.4
- github.com/hashicorp/hcl@v1.0.0
- github.com/hdevalence/ed25519consensus@v0.0.0-20210204194344-59a8610d2b87
- github.com/improbable-eng/grpc-web@v0.14.1
- github.com/klauspost/compress@v1.13.6
- github.com/lib/pq@v1.10.4
- github.com/libp2p/go-buffer-pool@v0.0.2
- github.com/magiconair/properties@v1.8.5
- github.com/mattn/go-isatty@v0.0.14
- github.com/matttproud/golang_protobuf_extensions@v1.0.2-0.20181231171920-c182affec369
- github.com/mimoo/StrobeGo@v0.0.0-20181016162300-f8f6d4d2b643
- github.com/minio/highwayhash@v1.0.2
- github.com/mitchellh/mapstructure@v1.4.3
- github.com/mtibben/percent@v0.2.1
- github.com/pelletier/go-toml@v1.9.4
- github.com/pkg/errors@v0.9.1
- github.com/pmezard/go-difflib@v1.0.0
- github.com/prometheus/client_golang@v1.12.1
- github.com/prometheus/client_model@v0.2.0
- github.com/prometheus/common@v0.32.1
- github.com/prometheus/procfs@v0.7.3
- github.com/rakyll/statik@v0.1.7
- github.com/rcrowley/go-metrics@v0.0.0-20200313005456-10cdbea86bc0
- github.com/regen-network/cosmos-proto@v0.3.1
- github.com/rs/cors@v1.8.2
- github.com/rs/zerolog@v1.26.0
- github.com/spf13/afero@v1.6.0
- github.com/spf13/cast@v1.4.1
- github.com/spf13/cobra@v1.3.0
- github.com/spf13/jwalterweatherman@v1.1.0
- github.com/spf13/pflag@v1.0.5
- github.com/spf13/viper@v1.10.1
- github.com/stretchr/testify@v1.7.0
- github.com/subosito/gotenv@v1.2.0
- github.com/syndtr/goleveldb@v1.0.1-0.20200815110645-5c35d600f0ca
- github.com/tendermint/btcd@v0.1.1
- github.com/tendermint/crypto@v0.0.0-20191022145703-50d29ede1e15
- github.com/tendermint/go-amino@v0.16.0
- github.com/tendermint/starport@v0.19.2
- github.com/tendermint/tendermint@v0.34.16
- github.com/tendermint/tm-db@v0.6.7
- golang.org/x/crypto@v0.0.0-20210915214749-c084706c2272
- golang.org/x/net@v0.0.0-20220127200216-cd36cc0744dd
- golang.org/x/sys@v0.0.0-20220114195835-da31bd327af9
- golang.org/x/term@v0.0.0-20210927222741-03fcf44c2211
- golang.org/x/text@v0.3.7
- google.golang.org/genproto@v0.0.0-20220519153652-3a47de7e79bd
- google.golang.org/grpc@v1.46.2 => google.golang.org/grpc@v1.33.2
- google.golang.org/protobuf@v1.28.0
- gopkg.in/ini.v1@v1.66.2
- gopkg.in/yaml.v2@v2.4.0
- gopkg.in/yaml.v3@v3.0.1
- nhooyr.io/websocket@v1.8.6
cosmos_sdk_version: v0.45.2
```

Linked LIbraries
```
aura@horace:~$ ldd go/bin/aurad
        linux-vdso.so.1 (0x00007ffee34e2000)
        libwasmvm.so => /validator-home-1/aura/go/pkg/mod/github.com/!cosm!wasm/wasmvm@v1.0.0-beta7/api/libwasmvm.so (0x00007f45ee200000)
        libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f45edfd8000)
        libgcc_s.so.1 => /lib/x86_64-linux-gnu/libgcc_s.so.1 (0x00007f45ee803000)
        librt.so.1 => /lib/x86_64-linux-gnu/librt.so.1 (0x00007f45ee7fe000)
        libpthread.so.0 => /lib/x86_64-linux-gnu/libpthread.so.0 (0x00007f45ee7f7000)
        libm.so.6 => /lib/x86_64-linux-gnu/libm.so.6 (0x00007f45edef1000)
        libdl.so.2 => /lib/x86_64-linux-gnu/libdl.so.2 (0x00007f45ee7f2000)
        /lib64/ld-linux-x86-64.so.2 (0x00007f45ee82c000)
```

**Comment by kienvc** [comment]
The error happened due to lack of wasm data when sync using state-sync mode
fix with patch version v0.3.2
